### PR TITLE
Cloud event blog update

### PR DESCRIPTION
**Pull request by abutch3r**
CloudEvents sdk-java 2.5.0 release included the new `cloudevents-http-restful-ws-jakarta` package for the first time which supports JakartaEE9+ making the build steps in the blog no longer required.

Remove build steps and just provide the pom snippet needed for CloudEvents

Improve image used to show Scale to 0 to better show multi instances and better resolution/size based on feedback

minor doc improvements

### CODE REVIEW

**Review by gljrobinson — DISMISSED** [review]
LGTM

File: posts/2022-10-17-microprofile-serverless-ibm-code-engine.adoc [hunk 1]
```diff
@@ -111,38 +115,32 @@ As part of creating your application on ICCE, you obtained the application URL f
...
-
-ICCE connects event providers such as https://cloud.ibm.com/docs/codeengine?topic=codeengine-working-kafkaevent-producer#setup-kafka-receiverapp[Kafka] or https://cloud.ibm.com/docs/codeengine?topic=codeengine-eventing-cosevent-producer#obstorage_ev_app[IBM Cloud Object Storage] to the application by using CloudEvents. These events are sent to a subscribed application as HTTP POST requests. Because the requests are in HTTP format, a RESTful application can receive these events without needing additional libraries and configuration.
+A common use case for serverless applications is to process events coming from non-HTTP sources, such as Kafka topics or Object Stores. Historically, to consume these events the application had to use the Cloud provider's SDK, which locks the application into that provider. This is where CloudEvents comes in, as it provides a cross-provider standard around which data can be transmitted and received by an application, improving portability and reducing large dependencies.
```
Comment (dmuelle) [inline posts/2022-10-17-microprofile-serverless-ibm-code-engine.adoc]:
```suggestion
A common use case for serverless applications is to process events coming from non-HTTP sources, such as Kafka topics or object stores. Historically, to consume these events, the application had to use the Cloud provider's SDK, which locks the application into that provider. This is where CloudEvents comes in. It provides a cross-provider standard around which applications can transmit and receive data, improving portability and reducing large dependencies.
```

File: posts/2022-10-17-microprofile-serverless-ibm-code-engine.adoc [hunk 2]
```diff
@@ -111,38 +115,32 @@ As part of creating your application on ICCE, you obtained the application URL f
...
 
-The CloudEvent 2.5.0 release, the first release of the `restful-ws-jakarta` package that we use to add CloudEvent support, is not yet publicly available at the time of this writing. As such, you must build it locally.
+ICCE connects event providers such as https://cloud.ibm.com/docs/codeengine?topic=codeengine-working-kafkaevent-producer#setup-kafka-receiverapp[IBM Cloud Event Streams] or https://cloud.ibm.com/docs/codeengine?topic=codeengine-eventing-cosevent-producer#obstorage_ev_app[IBM Cloud Object Storage] to the application by using CloudEvents. These events are sent to a subscribed application as HTTP POST requests. Because the requests are in HTTP format, a RESTful application can receive these events without needing additional libraries and configuration.
 
-Complete the following steps to build the `restful-ws-jakarta` package and its pre-requisites.
+To be able to process CloudEvents in Open Liberty add the CloudEvents `restful-ws-jakarta` library to the application by adding this dependency to your `pom.xml`:
```
Comment (dmuelle) [inline posts/2022-10-17-microprofile-serverless-ibm-code-engine.adoc]:
```suggestion
To be able to process CloudEvents in Open Liberty, add the CloudEvents `restful-ws-jakarta` library to the application by adding the following dependency to your `pom.xml`:
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
a few minor ID suggestions

**Review by dmuelle — APPROVED** [review]

**Pull request merged**